client/verta/tests/registry/model_version/test_set_schema.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this test module redundant since we already have tests covering `deployable_entity`?

Reply:
oop, yeah, forgot to clean up